x/staking/client/cli/utils.go

```diff
@@ -1,12 +1,77 @@
 package cli
 
 import (
+	"encoding/json"
 	"errors"
+	"fmt"
+	"os"
 
+	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
+// validator struct to define the fields of the validator
+type validator struct {
```

Comment:
I think we should have two validator structs, defined inside `parseValidatorJSON` and another here, with the correct types directly.

Then the validate function should do more than checking for emptyness.